src/components/Footer/index.js

```diff
@@ -45,8 +46,45 @@ const LinkContainer = ({ children }) => {
 };
 
 const Footer = () => {
+  //fix: sync this with localStorage
+  const [cookieBanner, setCookieBanner] = useState(true);
```

Comment:
@manangouhari  By default it's true. I think that's why there's a flicker even after closing the pop-up.

To solve the bug. Below logic would help.

```javascript
const [cookieBanner, setCookieBanner] = useState(false); // keep it false by default


  useEffect(() => {
    if (typeof window !== 'undefined') {
      let banner = localStorage.getItem('cookieBanner');
// show if banner has never been closed before.
      if (banner === null) {
        setCookieBanner(true);
      }
    }
  }, []);
```